python/cugraph/cugraph/community/__init__.py

```diff
@@ -55,10 +55,10 @@ def __call__(self, *args, **kwargs):
 else:
     __kuvs = ".".join([str(n) for n in __ktruss_unsupported_cuda_version])
     k_truss = __UnsupportedModule(
-        NotImplementedError("k_truss is not currently supported in CUDA"
+        NotImplementedError("k_truss is not currently supported in CUDA_VER"
```

Comment:
@rlratzel - Leaving this for you.  Do we want to change the text of an error message to say CUDA_VER rather than CUDA?

Reply:
No, `CUDA` is used here as the acronym in a user-facing error message, and is not a shell variable to be replaced.